src/components/booksList/BooksList.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC } from "react";
+import "./BooksList.css";
+import { IBook } from "../../interfaces";
+import Book from "../book/Book";
+import Pagination from "../pagination/Pagination";
+import { useDispatch, useSelector } from "react-redux";
+import { unselectAllBooks } from "../../store/slices/selectedBooksSlice";
+import {
+  selectBook,
+  unselectBook,
+} from "../../store/slices/selectedBooksSlice";
+import { RootState } from "../../store/store";
+
+type Props = {
+  totalPages: number;
+  currentPage: number;
+};
+
+const BooksList: FC<Props> = ({ totalPages, currentPage }) => {
+  const list = useSelector(
+    (state: RootState) => state.currentPageItems.currentPageItems,
+  );
```

Comment:
```suggestion
  const selectCurrentPageItems =  (state: RootState) => state.currentPageItems.currentPageItems
  
  const list = useSelector(selectCurrentPageItems);
```
и в отдельный файл их. прям в папку слайса. я про selectCurrentPageItems.  сверху писал в комменте про структуру
называешь их selectТо, selectСё